types/ramda/index.d.ts

```diff
@@ -842,7 +842,7 @@ declare namespace R {
          * Returns a list containing the names of all the enumerable own
          * properties of the supplied object.
          */
-        keys<T>(x: T): string[];
+        keys<T>(x: T): Array<keyof T>;
```

Comment:
This is technically not correct since types are open-ended; the known set of properties on an object may be a subset of those that exist at runtime.